configure

```diff
@@ -944,6 +946,8 @@ Optional Libraries:
   --with-opengl-prefix=DIR Prefix where OpenGL (ES) is installed (optional)
   --disable-opengl         disable OpenGL (ES) support [autodetect]
 
+  --with-dispmanx          enable dispmanx (Raspberry Pi native 2D API) rendering 
```

Comment:
This switch doesn't seem to be handled? I.e. it should be --enable-dispmanx? (How does this flag affect anything btw.? Wouldn't dispmanx still only be used when the backend/host is raspberrypi?)